Frends.Web/Web.cs

```diff
@@ -126,11 +125,46 @@ public class Options : IEquatable<Options>
         [UIHint(nameof(Frends.Web.Authentication), "", Authentication.OAuth)]
         public string Token { get; set; }
 
+        /// <summary>
+        /// Specifies where the Client Certificate should be loaded from.
+        /// </summary>
+        [UIHint(nameof(Frends.Web.Authentication), "", Authentication.ClientCertificate)]
+        [DefaultValue(CertificateSource.CertificateStore)]
+        public CertificateSource ClientCertificateSource { get; set; }
+
+        /// <summary>
+        /// Path to the Client Certificate when using a file as the Certificate Source, pfx (pkcs12) files are recommended. For other supported formats, see
+        /// https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509certificate2collection.import?view=netframework-4.7.1
+        /// </summary>
+        [UIHint(nameof(Frends.Web.Authentication), "", Authentication.ClientCertificate)]
```

Comment:
Is this supposed to be checking on the CertificateSource instead?  Could we hide the extra inputs that way? Or does the editor only support one layer of UIHints?

Reply:
Yes, sadly the editor/task loader supports only one UIHint per property. So if we used CertificateSource, it the certificate options would be shown when ClientCertificate authentication is not used. I thought it is clearer this way as client certificates are not used that often